src/js/configs.js

```diff
@@ -194,6 +194,16 @@ tomcat: {
     tls13: '8.0.0',
     usesOpenssl: false,
   },
+  kubelet: {
+    cipherFormat: 'go',
+    highlighter: 'yaml',
+    tls13: '1.14',
```

Comment:
I can only reference TLS 1.3 support since v1.19 — could you please provide the resource to confirm v1.14 is correct?

Reply:
@atombrella +please order the items alphabetically — tls13 comes before usesOpenssl…

Reply:
(actually you have to place the whole `kubelet: { … }` entry in its correct position alphabetically, otherwise it would get rendered between Tomcat and Traefik…)

Reply:
You can see the current build of the PR here: https://mje-k8s--mozsslconf-dev.netlify.app/#server=kubelet to play around with it and see the results.